src/main/java/frc/trigon/robot/subsystems/arm/simulationarm/SimulationArmIO.java

```diff
@@ -0,0 +1,98 @@
+package frc.trigon.robot.subsystems.arm.simulationarm;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.arm.ArmConstants;
+import frc.trigon.robot.subsystems.arm.ArmIO;
+import frc.trigon.robot.subsystems.arm.ArmInputsAutoLogged;
+
+public class SimulationArmIO extends ArmIO {
+    private final SingleJointedArmSim angleMotor = SimulationArmConstants.ANGLE_MOTOR;
+    private final ElevatorSim elevatorMotor = SimulationArmConstants.ELEVATOR_MOTOR;
+
+    private double angleMotorVoltage, elevatorMotorVoltage;
+
+    @Override
+    protected void updateInputs(ArmInputsAutoLogged inputs) {
+        angleMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+        elevatorMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+
+        inputs.angleMotorVoltage = angleMotorVoltage;
```

Comment:
here, check if the robot is disabled (DriverStation.isDisabled()). If it is, stop the motors. You need to do this since in the real robot this will happen, but not the sim.